packages/fulfillment/src/utils/utils.ts

```diff
@@ -0,0 +1,144 @@
+import {isString, MedusaError, pickValueFromObject} from "@medusajs/utils"
+
+export type Rule = {
+  attribute: string
+  operator: RuleOperator
+  value: string | string[]
+}
+
+export enum RuleOperator {
```

Comment:
Should this be part of the top-level utils?

Reply:
at somepoint, all those utils might be moved higher, for now I am constraining it to the module to not create any confusion. But we will have to discuss something about all the module having the same approach to standardizing it and reuse those utils

Reply:
I can add a todo if you think it could be good to have?

Reply:
Yeah at least a todo would be nice 👍